scripts/bsd/openbsd-prep-postgres.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+pkg_add -uvI && \
```

Comment:
This updates all the already installed packages, right? If so, that should probably be part of the base image script instead.